plugins/azure/appConfigurations/appConfigurationManagedIdentity.spec.js

```diff
@@ -0,0 +1,118 @@
+var expect = require('chai').expect;
+var appConfigurationManagedIdentity = require('./appConfigurationManagedIdentity.js');
+
+const appConfigurations = [
+    {
+        "type": "Microsoft.AppConfiguration/configurationStores",
+        "location": "eastus",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "creationDate": "2023-12-27T09:26:54+00:00",
+          "endpoint": "https://meerab-test-rg.azconfig.io",
+          "encryption": {
+            "keyVaultProperties": null
+          },
+          "privateEndpointConnections": null,
+          "publicNetworkAccess": "Disabled",
+          "disableLocalAuth": false,
+          "softDeleteRetentionInDays": 0,
+          "enablePurgeProtection": false
+        },
+        "id": "/subscriptions/26a1a07e-06dd-4892-92c9-e4996b0fc546/resourceGroups/meerab-rg/providers/Microsoft.AppConfiguration/configurationStores/meerab-test-rg",
+        "name": "meerab-test-rg",
+        "tags": {}
+    },
+    {
+        "type": "Microsoft.AppConfiguration/configurationStores",
+        "location": "eastus",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "creationDate": "2023-12-27T09:26:54+00:00",
+          "endpoint": "https://meerab-test-rg.azconfig.io",
+          "encryption": {
+            "keyVaultProperties": null
+          },
+          "privateEndpointConnections": null,
+          "publicNetworkAccess": "Disabled",
+          "disableLocalAuth": false,
+          "softDeleteRetentionInDays": 0,
+          "enablePurgeProtection": false
+        },
+        "id": "/subscriptions/26a1a07e-06dd-4892-92c9-e4996b0fc546/resourceGroups/meerab-rg/providers/Microsoft.AppConfiguration/configurationStores/meerab-test-rg",
```

Comment:
```suggestion
        "id": "/subscriptions/26a1a07e-06dd-4892-92c9-e4996b0fc546/resourceGroups/meerab-rg/providers/Microsoft.AppConfiguration/configurationStores/meerab-test-rg",
```
change subscription id to a random value